opentelemetry-api/src/opentelemetry/trace/__init__.py

```diff
@@ -139,6 +139,33 @@ class SpanKind(enum.Enum):
     CONSUMER = 4
 
 
+class SpanStatus:
+    """Represents the status of a finished Span """
+
+    def get_canonical_code(self) -> int:
+        """Gets the status CanonicalCode.
+        StatusCanonicalCode represents the canonical set of status codes of a finished Span, following the Standard GRPC codes
+        https://github.com/grpc/grpc/blob/master/doc/statuscodes.md
```

Comment:
We should link to the [OpenTelemetry spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/api-tracing.md#statuscanonicalcode) instead of gRPC spec (although the OpenTelemetry spec does link to gPRC).